### ISSUE TITLE
Consider fire_hydrant:type

### ISSUE THREAD

**Issue by rusty-snake** [organicmaps/organicmaps#8841]
**Is your feature request related to a problem? Please describe.**

[emergency=fire_hydrant](https://wiki.openstreetmap.org/wiki/Tag:emergency=fire_hydrant) is rendered unconditionally (on high zoom levels).

Especially `fire_hydrant:type=underground` has no use, has a missleading icon and "spams" the map.

**Describe the ideal solution**

Adhere to `fire_hydrant:type`, in detail

| `fire_hydrant:type=pillar` | `fire_hydrant:type=pipe` | `fire_hydrant:type=underground` |
| -------------------------- | ------------------------ | ------------------------------- |
| https://wiki.openstreetmap.org/wiki/File:Ofm_ofhydrant.jpg <br> This is what the current OM icon for fire hydrants looks like. <br> These objects are an orientation mark. <br> => Render them with the current icon. | https://wiki.openstreetmap.org/wiki/File:Hydrants_20130326_112938.JPG <br> => Render them, but with a better icon | Are not an orientation mark. Are as useful as `man_made=manhole`. <br> => Do not render them |

**Comment by RedAuburn** [comment]
agreed we should ignore some types, but is there really any benefit to rendering fire hydrants at all?

**Comment by biodranik** [comment]
Historically, MapsMe, and then Organic Maps were used by firefighters in some countries, and also in some cases of disaster responses when the Internet was not available.

As we're already drawing hydrants, it doesn't harm to add more tiny details there if it won't take resources from our team focused on high-priority issues. At least hiding the underground type should be easy. @vng is there a built-in filter mechanism where we can just delete/skip some unnecessary types/tag combinations?

**Comment by rusty-snake** [comment]
- Pillar can surf as a minor orientation mark. However it might be to minor to render it.
- For firefighters/disaster a specific renderer/app/overlay (all to much work for OM I think) is better than a high zoom level detail I guess. Such an rendere/overlay/app can show more details like diameter for hydrants and highlight other disaster POI.
  - Is there a possibility to keep fire_hydrants in the search/"show on map" while hiding in "normal" mode?

**Comment by biodranik** [comment]
Leaving hydrants as is works and do not require any changes )

**Comment by hemanggs** [comment]
This issue can be closed  , i  guess.

**Comment by tobru** [comment]
I'd love to see more details about `emergency=fire_hydrant`. Especially `fire_hydrant:type`, `fire_hydrant:pressure`, `couplings:diameters=*` and `ref=*`. As an active firefighter and regular user of Organic Maps, that would help tremendously. 

Recently, I put significant effort into having _all_ the fire hydrants in our community mapped in OSM, with every detail. It would be truly helpful to see this data in Organic Maps.
